### ISSUE TITLE
A bug in ModelNBody.cpp

### ISSUE THREAD

**Issue by DJLinux** [beltoforion/Barnes-Hut-Simulator#2]
In file: ModelNBody.cpp  method: ModelNBody::Init()

m_center.y += st.x * st_aux.mass; 
must be:
m_center.y += st.y * st_aux.mass; 

DJ

**Comment by beltoforion** [comment]
Yes that sounds plausible. But it doesn't really matter since this setup is symmetric. Nevertheless good find, thank you for reporting. Took only three years for someone to spot it.
